spring-boot/admin/src/main/resources/templates/admin/fragments/api-keys/consumer/list.html

```diff
@@ -97,6 +77,13 @@ <h2 th:text="#{admin.api-keys-consumer.subtitle}">[Post Administration</h2>
                                         <td th:text=" ${apiKey.keyValue}">Key Value</td>
                                         <td th:if="${testService == null}" class="a-center " th:text="${apiKey.created != null} ? ${#dates.format(apiKey.created, 'yyyy-MM-dd')} : ''">Created</td>
                                         <td th:if="${testService == null}" th:id="|@{item}${apiKey.id}@{ValidUntil}|" th:attr="validUntil=${#dates.format(apiKey.validUntil, 'yyyy-MM-dd')}" class="a-center " th:text="${apiKey.validUntil != null} ? ${#dates.format(apiKey.validUntil, 'yyyy-MM-dd')}: ''">Valid until</td>
+                                        <td class="right">
+                                            <button th:onclick="|@{location.href='/admin/api-keys/}${apiKey.id}@{'}|" ><i class="fa fa-search-plus"/></button>
+                                            <button th:if="${apiKey.serviceApi.serviceAccessPermissionPolicy.displayStringShort == 'Public'}"
```

Comment:
The policy can change over time, we should discuss if this could lead to problems.

Reply:
General comment: display strings should not be used for program logic. Use enum values instead.

Reply:
Done.

Reply:
Just about possible policy changes:
- If a previously **private** service becomes **public**, the user could delete an API key that was created by another user (should not be a big deal as long as the metrics/statistics are not deleted).
- If a previously **public** service becomes **private**, the user will not be able anymore to delete an API key that this user created. This could be annoying, but will rarely happen.

Reply:
Done. The feature works as specified by you.